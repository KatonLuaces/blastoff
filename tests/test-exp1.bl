M = [1, 2; 3, 4];
b = 2;
print(toString(M^b));