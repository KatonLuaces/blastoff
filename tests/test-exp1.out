7 10 
15 22 